loader/CMakeLists.txt

```diff
@@ -229,21 +229,14 @@ if(WIN32)
                 $<TARGET_OBJECTS:loader-unknown-chain>
                 ${CMAKE_CURRENT_SOURCE_DIR}/vulkan-1.def
                 ${CMAKE_CURRENT_SOURCE_DIR}/loader.rc)
-    if(MSVC)
-    set_target_properties(vulkan
-                          PROPERTIES
-                          #LINK_FLAGS_DEBUG "/ignore:4098"
-                                     OUTPUT_NAME
-                                     vulkan-1
-                                     PREFIX
-                                     "")
-    else()
+
+    # when adding the suffix the import and runtime library names must be consistent
+    # mingw: libvulkan-1.dll.a / libvulkan-1.dll
```

Comment:
Doesn't it defeat the purprose of https://github.com/KhronosGroup/Vulkan-Loader/pull/523?